towerlib/entities/job.py

```diff
@@ -57,12 +57,14 @@
 
 
 class Job:  # pylint: disable=too-few-public-methods
-    """Job factory to handle the different jod types returned."""
+    """Job factory to handle the different job types returned."""
 
     def __new__(cls, tower_instance, data):
         entity_type = data.get('type')
         if entity_type == 'job':
             obj = JobRun(tower_instance, data)
+        if entity_type == 'workflow_job':
```

Comment:
Since there is already a conditional checking the entity_type this should be an elif.

Reply:
of course.